### PR TITLE
[Rule Tuning] Q2 Linux DR Tuning - Part 5

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
Several Linux DR tunings

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: Tuning - Guidelines

These guidelines serve as a reminder set of considerations when tuning an existing rule.

### Documentation and Context

- [ ] Detailed description of the suggested changes.
- [ ] Provide example JSON data or screenshots.
- [ ] Provide evidence of reducing benign events mistakenly identified as threats (False Positives).
- [ ] Provide evidence of enhancing detection of true threats that were previously missed (False Negatives).
- [ ] Provide evidence of optimizing resource consumption and execution time of detection rules (Performance).
- [ ] Provide evidence of specific environment factors influencing customized rule tuning (Contextual Tuning).
- [ ] Provide evidence of improvements made by modifying sensitivity by changing alert triggering thresholds (Threshold Adjustments).
- [ ] Provide evidence of refining rules to better detect deviations from typical behavior (Behavioral Tuning).
- [ ] Provide evidence of improvements of adjusting rules based on time-based patterns (Temporal Tuning).
- [ ] Provide reasoning of adjusting priority or severity levels of alerts (Severity Tuning).
- [ ] Provide evidence of improving quality integrity of our data used by detection rules (Data Quality).
- [ ] Ensure the tuning includes necessary updates to the release documentation and versioning.

### Rule Metadata Checks

- [ ] `updated_date` matches the date of tuning PR merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive. Review to ensure the original intent of the rule is maintained.

### Testing and Validation

- [ ] Validate that the tuned rule's performance is satisfactory and does not negatively impact the stack.
- [ ] Ensure that the tuned rule has a low false positive rate.

File: rules/linux/persistence_rpm_package_installation_from_unusual_parent.toml [hunk 1]
```diff
@@ -21,12 +21,16 @@ name = "RPM Package Installed by Unusual Parent Process"
 risk_score = 21
 rule_id = "8cc72fa3-70ae-4ea1-bee2-8e6aaf3c1fcf"
 setup = """## Setup
+
```
Comment (terrancedejesus) [inline rules/linux/persistence_rpm_package_installation_from_unusual_parent.toml:24]: I don't see any query adjustments here. If not the contents will still version bump the rule and display to the user that this rule has been updated.

File: rules/linux/persistence_systemd_scheduled_timer_created.toml [hunk 2]
```diff
@@ -176,6 +174,10 @@ file where host.os.type == "linux" and event.action in ("rename", "creation") an
     "/opt/puppetlabs/puppet/bin/ruby", "/usr/libexec/platform-python", "/kaniko/kaniko-executor",
     "/usr/local/bin/dockerd", "/usr/bin/podman", "/bin/install", "/proc/self/exe"
   ) or
+  process.name like (
+    "python*", "crio", "apt-get", "install", "snapd", "cloudflared", "sshd", "convert-usrmerge", "docker-init",
+    "google_metadata_script_runner"
```
Comment (terrancedejesus) [inline rules/linux/persistence_systemd_scheduled_timer_created.toml:179]: google_metadata_script_runner is the equivalent of AWS's user-data scripts. Typically bash scripts meant to run at startup for a cloud instance. Startup scripts are a great persistence mechanism in cloud instances so while we ignore here, we could potentially look at a rule to identify anomalous cloud instance startup scripts later on 🤷🏽

**Review by terrancedejesus — APPROVED** [review]

File: rules/linux/persistence_systemd_scheduled_timer_created.toml (hunk 2 above)
Comment (Aegrah) [reply rules/linux/persistence_systemd_scheduled_timer_created.toml:179]: I could remove those from the tuning. Those weren't that big in terms of FPs. If this tuning introduces cloud evasion opportunities I will get it out of this and other rules!

File: rules/linux/persistence_rpm_package_installation_from_unusual_parent.toml (hunk 1 above)
Comment (Aegrah) [reply rules/linux/persistence_rpm_package_installation_from_unusual_parent.toml]: I reverted the change to that rule. I will fix the setup guide in a separate bulk PR.

**Review by shashank-elastic — APPROVED** [review]

**Pull request merged**